docs/specs/markdown.yml

```diff
@@ -628,15 +628,17 @@ outputs:
 # Liquid template file missing should not crash
 inputs: 
   docfx.yml: |
-    outputJson: false
+    outputType: html
+    outputUrlType: pretty
   docs/a.md: Hello `docfx`! 
 outputs: 
   docs/a/index.html:
 ---
 # Source file name ending with spaces should not crash
 inputs:
   docfx.yml: |
-    outputJson: false
+    outputType: html
+    outputUrlType: pretty
```

Comment:
what does the output look like for `index.md` when `pretty`?

Reply:
handled now, see the first test case

---
In reply to: [413550133](https://github.com/dotnet/docfx/pull/5806#discussion_r413550133) [](ancestors = 413550133)